### DISCUSSION TITLE
Gaia-X Data Contract Service support in EDC

### DISCUSSION THREAD

**Discussion by Yazan-aljaloudi** [eclipse-edc/.github#32] [Q&A]
in some resources it is stated that Eclipse Data Space Connector (EDC) was working on implementing support for Gaia-X Data Contract Service (DCS).

if so where can i find more information on this?

**Comment by jimmarino** [comment]
Currently, EDC is working on implementing the IDS Dataspace Protocol specifications, which are being announced this week.

Many of us involved in EDC would also be interested in Gaia-X DCS. However, there appears to be no public specification or technical details to evaluate. Also, it is unclear how it relates to or differs from IDS.  Maybe someone can shed light on that?
